public/components/event_analytics/explorer/explorer.scss

```diff
@@ -15,7 +15,7 @@
 }
 
 .mainContentTabs .euiResizableContainer {
-  height: calc(100vh - 298px);
+  height: calc(100vh - 194px);
```

Comment:
did you test event page and possible resizing of the the page?

Reply:
Yes, this value is derived from the size of all of the components on top (such as search bar, etc)